packages/eslint-plugin/.eslintrc.js

```diff
@@ -0,0 +1,16 @@
+"use strict";
+
+module.exports = {
+  "parserOptions": {
+    "ecmaVersion": "latest"
+  },
+  root: true,
+  extends: [
```

Comment:
Why does it needed?

Reply:
good part is eslint rule for eslint plugins)